week-2/E-conditionals/exercise.js

```diff
@@ -10,8 +10,12 @@ var role = "mentor";
 
 var name2 = "Harun";
 var role2 = "student";
-
-
+ if (role == "student"){
+   return name;
```

Comment:
Hmm...you are not inside a function. What does this return do? Does your code run and do what is expected?